### ISSUE TITLE
ufront --setup fails

### ISSUE THREAD

**Issue by matthewswallace** [ufront/ufront#27]
when running sudo haxelib ufront-tool --setup I get the following errors. 

```
/usr/local/lib/haxe/lib/ufront-tool/1,0,0/src/uftool/SysUtil.hx:99: characters 12-17 : Command `haxelib path ufront` failed.
/usr/local/lib/haxe/lib/ufront-tool/1,0,0/src/uftool/SysUtil.hx:99: characters 12-17 : Exit code: 1
/usr/local/lib/haxe/lib/ufront-tool/1,0,0/src/uftool/SysUtil.hx:99: characters 12-17 : Stdout: Error: Library ufront is not installed : run 'haxelib install ufront'
/usr/local/lib/haxe/lib/ufront-tool/1,0,0/src/uftool/SysUtil.hx:99: characters 12-17 :
/usr/local/lib/haxe/lib/ufront-tool/1,0,0/src/uftool/SysUtil.hx:99: characters 12-17 : Stderr:
/usr/local/lib/haxe/lib/ufront-tool/1,0,0/src/uftool/UFTool.hx:178: characters 18-74 : Called from
?:1: characters 1048575-8796094070782 : Called from
```

**Comment by kevinresol** [comment]
I think you missed `run`
`sudo haxelib run ufront-tool --setup`

**Comment by matthewswallace** [comment]
👍

**Comment by Anheurystics** [comment]
I'm also getting an error when trying to run setup, on Windows 10, it tells me that the current directory's path is an unknown argument.

**Comment by kevinresol** [comment]
This should be fixed in 1.0.1
Closing for now. Feel free to open another issue if there are still problems.
